templates/profile/view_profile.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'base/header.html' %}
+{% block content %}
+<div class="my-account">
+    <div class="container-fluid">
+        <div class="row">
+        <div class="col-md-9">
+            <div class="tab-content">
+                <div id="address-tab" role="tabpanel" >
+                <h1>{{ object.username }}</h1><br><br>
+                <div class="row">
+                    <div class="col-md-9">
+                        <h5>{{ user.first_name }}</h5>
+                        <h5>{{ user.last_name }}</h5>
+                        <p>Phone: {{ user.phone_no }}</p><br>
+                        <button class="btn">Edit Profile</button><br><br>
+                    </div>
+                </div>
+                    <h2>Address</h2><br>
+                    {% for address in addresses %}
+                    <div class="row">
+                    <div class="col-md-6">
+                            <h3>
+                                {% if address.is_primary %}
+                                    Default Address
+                                {% elif not other_address_displayed %}
```

Comment:
not getting this condition @rinkishivani

Reply:
changed elif to else